apps/server/src/routes/files/index.ts

```
@@ -29,11 +29,19 @@ filesRouter.get("/", async (c: Context) => {
 	}
 
-	// * The GoogleDriveProvider will be replaced by a general provider in the future
-	const files = await new GoogleDriveProvider(accessToken).listFiles();
-	if (!files) {
-		return c.json<FileOperationResponse>({ success: false, message: "Files not found" }, 404);
-	}
+	// Get URL params
+	const url = new URL(c.req.url);
+	const query = url.searchParams.get("q") || "";
+	const parents = url.searchParams.get("parents");
+	const pageSize = parseInt(url.searchParams.get("pageSize") || "20");
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)